cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp

```diff
@@ -207,15 +207,31 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
     {
         namespace py = pybind11; // NOLINT(misc-unused-alias-decls)
 
+#define def_arithmetic(FUNCNAME) \
+    def(#FUNCNAME, [](wrapped_type & self, wrapped_type const & other) { return wrapped_type::FUNCNAME(self, other); })
+
         (*this)
             .def("min", &wrapped_type::min)
             .def("max", &wrapped_type::max)
             .def("sum", &wrapped_type::sum)
             .def("abs", &wrapped_type::abs)
+            .def_arithmetic(add)
```

Comment:
Is it correct that not using the macro, the lines will be like:

```cpp
.def("add", &wrapped_type::add)
```

If so, don't use the macro.  Simply spelling out the wrapping code is more readable and maintainable.